rabbitmq/check.py

```diff
@@ -337,6 +328,20 @@ def _get_metrics(self, data, object_type, custom_tags):
                     self.log.debug("Caught ValueError for %s %s = %s  with tags: %s" % (
                         METRIC_SUFFIX[object_type], attribute, value, tags))
 
+    def _get_queue_bindings_metrics(self, data, instance, object_type):
+        base_url, _, _, auth, ssl_verify, custom_tags = self._get_config(instance)
+        instance_proxy = self.get_instance_proxy(instance, base_url)
+
+        for item in data:
+            vhost = item['vhost']
+            tags = self._get_tags(item, object_type, custom_tags)
+            url = '{}/{}/{}/bindings'.format(QUEUE_TYPE, urllib.quote_plus(vhost), item['name'])
+            self.log.info('url: {}'.format(urlparse.urljoin(base_url, url)))
```

Comment:
I don't think it's very useful to log the URL. If we want to do it we should use debug and add more information about what URL we are logging.